packages/zowe-explorer/src/trees/dataset/DatasetTree.ts

```diff
@@ -112,16 +112,25 @@ export class DatasetTree extends ZoweTreeProvider<IZoweDatasetTreeNode> implemen
         recursiveCall?: boolean
     ): Promise<void> {
         const destinationInfo = FsAbstractUtils.getInfoForUri(destUri, Profiles.getInstance());
-        let dsname = destUri.path.substring(destinationInfo.slashAfterProfilePos);
+        const sourceInfo = FsAbstractUtils.getInfoForUri(sourceUri, Profiles.getInstance());
+        let dsname = destUri.path.substring(destinationInfo.slashAfterProfilePos + 1);
         if (SharedContext.isPds(sourceNode)) {
             if (!DatasetFSProvider.instance.exists(destUri)) {
-                // create a PDS on remote
+                // find source PDS attributes and make destination PDS have same attributes
+                const sourceAttributesResponse = await ZoweExplorerApiRegister.getMvsApi(sourceInfo.profile).dataSet(dsname, {
+                    attributes: true,
+                    responseTimeout: sourceInfo.profile?.profile?.responseTimeout,
+                });
+                const { dsname: dsnameSource, ...rest } = sourceAttributesResponse.apiResponse.items[0];
+
+                // create a PDS on remote with the same attributes as source PDS
+                const transformedAttrs = (zosfiles.Copy as any).generateDatasetOptions({}, rest);
```

Comment:
All the data set attributes appear to be replicated correctly except for size which increases for me:
**Original PDS (size=15)**
<img width="330" alt="Image" src="https://github.com/user-attachments/assets/5cb603b3-7ca4-4de3-a132-882e2ebcb79d" />

**Moved PDS (size=225)**
<img width="330" alt="Image" src="https://github.com/user-attachments/assets/5b591845-bdcd-48d5-bfca-0649529396fb" />

Reply:
fixed!